lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -126,6 +126,7 @@ sub create_from_settings {
     my $job_settings  = $schema->resultset('JobSettings');
     my $txn_guard     = $result_source->storage->txn_scope_guard;
 
+    die 'The PUBLISH_HDD_1 cannot include / in value' if ($settings{PUBLISH_HDD_1} && $settings{PUBLISH_HDD_1} =~ /\//);
```

Comment:
this again applies only to the *first* specified value. Don't forget about `PUBLISH_HDD_2`, `STORE_HDD_1`, etc.